app/controllers/workflows_controller.rb

```diff
@@ -73,7 +90,23 @@ def workflow_params
     params.require(:workflow).permit(
       :public_extracts,
       :public_reductions,
+      :rules_applied,
       webhooks_config: {},
     )
   end
+
+  def heartbeat
+    @heartbeat = {
```

Comment:
Could this be extracted to a (non-DB) model instead, and respond_with that?

Reply:
Yeah, I can do that. I'd just put `@heartbeat` into the context probably rather than responding with it since it's not part of the path